.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,59 @@
+name: Bug Report
+description: Report incorrect behavior in the shap library
+title: "BUG: "
+labels: ["bug"]
+
+body:
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Shap version checks
+      options:
+        - label: >
+            I have checked that this issue has not already been reported.
+          required: true
+        - label: >
+            I have confirmed this bug exists on the
+            [latest version](https://github.com/slundberg/shap/releases) of shap.
+          required: true
+        # TODO: Link to instructions to install development version
+        - label: >
+            I have confirmed this bug exists on the master branch of shap.
+  - type: textarea
+    id: example
+    attributes:
+      label: Reproducible Example
+      description: >
+        Please follow [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) on how to
```

Comment:
The link can be updated to `https://matthewrocklin.com/minimal-bug-reports` to save on one level of redirection.